src/test/rpc/Book_test.cpp

```diff
@@ -800,6 +800,460 @@ class Book_test : public beast::unit_test::suite
             books)[jss::status] == "success");
     }
 
+    void testTrackOffers(void)
+    {
```

Comment:
well, I misunderstood your original comment - I thought you were referring to the actual method name, not the use of the `testcase()` function. Now that I understand, I will add it - although I have a distinct memory of some dissent around the broad use of `testcase()` precisely because it lengthens (clutters?) the output from the unit test runner.  The convention within this particular file seems to be to use it, so I will stick with that, but I'm not sure there is broad agreement on using it general. Sounds like we need a vote or an executive decision ;)